taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.contrib.auth import get_user_model
```

Comment:
Instead of using `get_user_model().objects.count()`, consider importing the User model directly from your models file if it's defined there. This can make your code a bit cleaner. You would then use `User.objects.count()`. If the User model isn't defined in your models file, ignore this suggestion.